lib/src/model/common/service/sound_service.dart

```diff
@@ -93,6 +94,7 @@ class SoundService {
               _currentStream!.$2 == Sound.capture)) {
         await _pool.stop(_currentStream!.$1);
       }
+      await _pool.setVolume(soundId: soundId, volume: volume);
```

Comment:
Play should have the lowest latency possible since we're playing short sounds very fast during game or replay.

So volume should not be set here.

Instead keep a reference to `masterVolume` in `SoundService` and have a `setVolume` method which loops over all loaded sound to change their volume one by one.

If needed at some point we could even tune the volume of each sound individually.

Reply:
I've updated the implementation now, hopefully it's what you were looking for.